core/arch/arm/kernel/thread_spmc.c

```diff
@@ -1528,6 +1694,8 @@ static TEE_Result spmc_init(void)
 	my_endpoint_id = spmc_get_id();
 	DMSG("My endpoint ID %#x", my_endpoint_id);
 
+	nw_rxtx.ffa_vers = INITIAL_NW_FFA_VERSION;
```

Comment:
The `nw_rxtx` here is actually the RXTX buffer between OP-TEE S-EL1 SP and the S-EL2 SPMC, it's not related to the NWd right?

Reply:
Right, I'll drop the `NW_` part.